package/main/daml/Daml.Finance.Interface.Types.Common/daml.yaml

```diff
@@ -1,10 +1,10 @@
 # Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-sdk-version: 2.7.0
+sdk-version: 2.7.4
 name: daml-finance-interface-types-common
 source: daml
-version: 1.0.99.20231115.1
+version: 2.99.0.20231116.1
```

Comment:
Shouldn't this one be `1.99`?

Reply:
yes, target should be 2.0.0 as in the changelog. I fixed it, thanks!